### ISSUE TITLE
Build failure with GHC 7.10

### ISSUE THREAD

**Issue by snoyberg** [yogsototh/holy-project#5]
```
Unpacking to holy-project-0.1.1.1/
Resolving dependencies...
Configuring holy-project-0.1.1.1...
Building holy-project-0.1.1.1...
Preprocessing library holy-project-0.1.1.1...
[1 of 6] Compiling Paths_holy_project ( dist/build/autogen/Paths_holy_project.hs, dist/build/Paths_holy_project.o )
[2 of 6] Compiling HolyProject.MontyPython ( src/HolyProject/MontyPython.hs, dist/build/HolyProject/MontyPython.o )
[3 of 6] Compiling HolyProject.GithubAPI ( src/HolyProject/GithubAPI.hs, dist/build/HolyProject/GithubAPI.o )
[4 of 6] Compiling HolyProject.StringUtils ( src/HolyProject/StringUtils.hs, dist/build/HolyProject/StringUtils.o )
[5 of 6] Compiling HolyProject.GitConfig ( src/HolyProject/GitConfig.hs, dist/build/HolyProject/GitConfig.o )
[6 of 6] Compiling HolyProject      ( src/HolyProject.hs, dist/build/HolyProject.o )
In-place registering holy-project-0.1.1.1...
Preprocessing executable 'holy-project' for holy-project-0.1.1.1...
[1 of 7] Compiling Paths_holy_project ( dist/build/autogen/Paths_holy_project.hs, dist/build/holy-project/holy-project-tmp/Paths_holy_project.o )
[2 of 7] Compiling HolyProject.MontyPython ( src/HolyProject/MontyPython.hs, dist/build/holy-project/holy-project-tmp/HolyProject/MontyPython.o )
[3 of 7] Compiling HolyProject.GitConfig ( src/HolyProject/GitConfig.hs, dist/build/holy-project/holy-project-tmp/HolyProject/GitConfig.o )
[4 of 7] Compiling HolyProject.GithubAPI ( src/HolyProject/GithubAPI.hs, dist/build/holy-project/holy-project-tmp/HolyProject/GithubAPI.o )
[5 of 7] Compiling HolyProject.StringUtils ( src/HolyProject/StringUtils.hs, dist/build/holy-project/holy-project-tmp/HolyProject/StringUtils.o )
[6 of 7] Compiling HolyProject      ( src/HolyProject.hs, dist/build/holy-project/holy-project-tmp/HolyProject.o )
[7 of 7] Compiling Main             ( src/Main.hs, dist/build/holy-project/holy-project-tmp/Main.o )
Linking dist/build/holy-project/holy-project ...
Preprocessing executable 'test-holy-project' for holy-project-0.1.1.1...
[1 of 3] Compiling HolyProject.GithubAPI.Test ( test/HolyProject/GithubAPI/Test.hs, dist/build/test-holy-project/test-holy-project-tmp/HolyProject/GithubAPI/Test.o ) [2 of 3] Compiling HolyProject.StringUtils.Test ( test/HolyProject/StringUtils/Test.hs, dist/build/test-holy-project/test-holy-project-tmp/HolyProject/StringUtils/Test.o )

test/HolyProject/StringUtils/Test.hs:14:1:
    Non type-variable argument in the constraint: SC.Testable IO a
    (Use FlexibleContexts to permit this)
    When checking that ‘tp’ has the inferred type
      tp :: forall a.
            (SC.Testable IO a, QC.Testable a) =>
            tasty-0.10.1.1:Test.Tasty.Core.TestName -> a -> TestTree
```

**Comment by yogsototh** [comment]
I'll fix it as soon as I'll get my hand on a computer. I'm on vacation for now. 

Envoyé de mon iPhone

**Comment by yogsototh** [comment]
I believe I've fixed the issue in the 0.2.0.0 version (uploaded on hackage), I'll update stackage ASAP.
